README.md

```diff
@@ -126,3 +126,72 @@ Now, for any new html page, we need to do the following:
 {% endblock %}
 
 ```
+
+## pre deployment to fly.io
+
+
+- Setting Up environmental variables
+
+  - Add `python-dotenv>=0.21.0` to requirements.txt
+  - Run `pip install -r requirements.txt`
+
+  -  create new .env file
+      add the following informations in it
+          SECRET_KEY
+          DATABASE_URL
+          DEBUG
+
+      NB for Secret key generate random variable of length 60
```

Comment:
Make this a script and just give command line isntructions that everyone can copy paste

Reply:
Ok, but It's one time job,

Reply:
Which makes it even nicer if the person can copy and paste into bash and be up and running in 15 seconds